templates/CONTRIBUTING.ejs.md

```diff
@@ -0,0 +1,52 @@
+# Contributing to <%= name %>
+
+Welcome to <%= name %>! We appreciate your interest in contributing. By participating in this project, you agree to abide by the [Code of Conduct](./CODE_OF_CONDUCT.md).
```

Comment:
This `<%= name %>`  is new to me! I haven't seen values being used in markdown before,

Reply:
It's not markdown. It's EJS. The engine at the core of the shoveler 

EJS let's you define variables. Populated by answers of prompts shown by enquirer. Then the engine replaces all placeholders in all templates by the variable value.

That's why they are templates. Without the engine that replaces placeholders like this one it would just be example or sample files.